app/components/views/Stake/index.tsx

```diff
@@ -349,21 +349,16 @@ export const Stake = (props: Props) => {
               </button>
             </div>
             <div className={styles.stakeInput}>
-              <Trans
-                id={inputPlaceholderMessage[view].id}
-                render={({ translation }) => (
```

Comment:
Did you check if the language switch still works here?
I remember that this render function was needed back then as the component did not re-render when the user switches the language and the placeholder remains in the former language